src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -1202,13 +1267,21 @@ int RadosBucket::set_logging_object_name(const std::string& obj_name,
                                ceph::real_time::clock::now(),
                                y,
                                no_change_attrs());
-  if (ret == -EEXIST) {
-    ldpp_dout(dpp, 20) << "INFO: race detected in initializing '" << obj_name_oid << "' with logging object name:'" << obj_name  << "'. ret = " << ret << dendl;
-  } else if (ret == -ECANCELED) {
-    ldpp_dout(dpp, 20) << "INFO: race detected in updating logging object name '" << obj_name << "' at '" << obj_name_oid << "'. ret = " << ret << dendl;
-  } else if (ret < 0) {
-    ldpp_dout(dpp, 1) << "ERROR: failed to set logging object name '" << obj_name << "' at '" << obj_name_oid << "'. ret = " << ret << dendl;
+  if (ret < 0) {
```

Comment:
did you change anything here or just the formatting?
(please try to avoid format change in code that you dont modify)